src/components/containerGradient/Display/Display.module.scss

```diff
@@ -32,6 +32,10 @@
     }
 
     border-right: unset !important;
+
+    &.vertical {
```

Comment:
asked in element

Reply:
Ability to display only one saber

<img width="428" alt="image" src="https://github.com/user-attachments/assets/22d33b2c-1b1b-481f-90d9-c358e13057a3">